space/space_resource.go

```diff
@@ -84,6 +87,11 @@ func (r *GormResourceRepository) Load(ctx context.Context, ID uuid.UUID) (*Resou
 	return &res, nil
 }
 
+// Exists returns true|false whether a space resource exists with a specific identifier
+func (m *GormResourceRepository) Exists(ctx context.Context, id string) (bool, error) {
+	return repository.Exists(ctx, m.db, Resource{}.TableName(), id)
```

Comment:
Fixed in https://github.com/almighty/almighty-core/pull/1389/commits/960ba3deddb447e3b51d11c251d1ee00c34799c3